google-cloud-pub-sub-grpc/src/main/scala/akka/stream/alpakka/googlecloud/pubsub/grpc/impl/AkkaGrpcSettings.scala

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.googlecloud.pubsub.grpc.impl
+
+import akka.actor.ActorSystem
+import akka.annotation.InternalApi
+import akka.grpc.GrpcClientSettings
+import akka.stream.alpakka.googlecloud.pubsub.grpc.PubSubConfig
+import com.typesafe.config.ConfigFactory
+
+/**
+ * Internal API
+ */
+@InternalApi private[grpc] object AkkaGrpcSettings {
+  def fromPubSubConfig(config: PubSubConfig)(implicit sys: ActorSystem): GrpcClientSettings = {
+    val sslConfig = config.rootCa.fold("") { rootCa =>
+      s"""
+      |ssl-config {
+      |  disabledKeyAlgorithms = []
+      |  trustManager = {
+      |    stores = [
+      |      { type = "PEM", path = "$rootCa", classpath = true }
+      |    ]
+      |  }
+      |}""".stripMargin
+    }
+
+    val akkaGrpcConfig = s"""
+      |host = "${config.host}"
+      |port = ${config.port}
+      |
+      |$sslConfig
+      |""".stripMargin
+
+    val settings =
+      GrpcClientSettings.fromConfig(
+        ConfigFactory
+          .parseString(akkaGrpcConfig)
+          .withFallback(sys.settings.config.getConfig("akka.grpc.client.\"*\""))
```

Comment:
Since we do not want to expose akka-grpc config settings, and the ssl-context loading part in   `GrpcClientSettings` [is private](https://github.com/akka/akka-grpc/blob/607599b0408c5e8bbdeb5bf59c07e9863c7ae566/runtime/src/main/scala/akka/grpc/GrpcClientSettings.scala#L123), we build `GrpcClientSettings` from Config and then set the remaining attributes (`useTls` and `callCredentials`) using setters.

//cc @raboof